helmfile/charts/rclone-sync/files/rclone.conf

```diff
@@ -0,0 +1,25 @@
+[{{ .Values.config.source.name }}]
+type = {{ .Values.config.source.type }}
+env_auth = false
```

Comment:
Looking at the documentation this option ([--s3-env-auth](https://rclone.org/s3/#s3-env-auth)) is false by default.